equinix/port_helpers.go

```diff
@@ -342,7 +342,7 @@ func portSanityChecks(cpr *ClientPortResource) error {
 	return nil
 }
 
-func portProperlyDestroyed(port *packngo.Port) error {
+func PortProperlyDestroyed(port *packngo.Port) error {
```

Comment:
This file is used in both tests and customer-facing code, so it's made public here, but otherwise doesn't change.